### ISSUE TITLE
Notification Module not working when in Cloud

### ISSUE THREAD

**Issue by ralphlenssen** [mendix/MxPushNotifications#14]
I have used the push notifications module version 3.0 which has been updated for Mendix 6.0. I have used this in modeller version 6.3.1.

When I run it locally the messages get sent without a problem. However on our cloud node, we get a java error:

APN: Message to <0b9ea091 d84e7daa dd6317d3 0c160fe1 2ee5a4f7 76dfb1eb 63dead22 6bca8614> failed: java.lang.RuntimeException: Invalid hex character: <

We use the exact same settings on the cloud as locally. The registration adres is the same and the APN service has the same certificate. The registration Id with the "<" is generated from somewhere deep in the module. Since the same input is used locally as in the cloud a invalid hex character error is very confusing. Any idears on what the real problem is would be very welcome.

What I have already tried:

Various ways of writing the registration adres
Restarting the APN service
Recreating the APN service
Checked that locally we run with emulate cloud settings

**Comment by simo101** [comment]
Hi Ralph, I'm not going to be able to look at this until Wednesday due to holidays. So I will try my best to take a look on Wednesday.

**Comment by ralphlenssen** [comment]
Would be very helpful if you could take a look today. Thank you in advanced

**Comment by simo101** [comment]
Hey @ralphlenssen I am fully booked today. So I am not going to be able to look at it today. Can you provide a test project with details on how I can replicate the functionality. This will save me time trying to fix this issue.

**Comment by ralphlenssen** [comment]
I have found out what the problem was.

We were generating the app that receives the notifications directly from Xcode. The provisioning profile that was used for the app only worked for my collegue, but not for my own device. Since I was testing it the second time from the cloud, I presumed that the difference layed in the environment.

Not sure if apple provides any feedback since APN is fire and forget, but if it does it would be helpfull to get error messages from this in the Mendix log. 

Another minor issue I found is that when I delete a APNSettings, the APNCertificate is not deleted. I fixed this myself by adding delete behaviour.
